arviz/plots/backends/bokeh/kdeplot.py

```diff
@@ -226,10 +229,15 @@ def plot_kde(
                 palette=colors,
                 **pcolormesh_kwargs
             )
+            glyphs.append(image)
             ax.x_range.range_padding = ax.y_range.range_padding = 0
 
     if backend_show(show):
         bkp.show(ax, toolbar_location="above")
+
+    if legend and label is not None:
+        return glyphs
```

Comment:
This is what is making the documentation build fail. Maybe bokeh plots can return `ax, (extra, elements)` or something like this? Maybe a return_glyphs argument only for internal use? @ahartikainen

Reply:
Yeah, maybe add some argument to bokeh function to return this for internal use. So it would default to false and can not be seen in the main function.

btw did you mean

`if legend and label is not None`

or

`if (legend is not None) and (label is not)` None

Reply:
I have done as @OriolAbril suggested. I just defined `return_glyphs` as `None` and returned it along the axes. If the user doesn't specify the legend, it will plot the graph without legend.